snuba/datasets/configuration/spans/entities/spans.yaml

```diff
@@ -128,7 +128,7 @@ storages:
             from_column_table: null
             from_column_name: sentry_tags
             to_nested_col_table: null
-            to_nested_col_name: tags
+            to_nested_col_name: sentry_tags
```

Comment:
We were fetching from `tags` instead of `sentry_tags` for the `sentry_tags` field.